parsl/executors/high_throughput/interchange.py

```diff
@@ -333,9 +324,14 @@ def start(self) -> None:
         # parent-process-inheritance problems.
         signal.signal(signal.SIGTERM, signal.SIG_DFL)
 
-        logger.info("Incoming ports bound")
+        logger.info("Starting main interchange method")
 
-        hub_channel = self._create_monitoring_channel()
+        if self.hub_address is not None and self.hub_zmq_port is not None:
+            logger.debug("Creating monitoring radio")
+            monitoring_radio = ZMQRadioSender(self.hub_address, self.hub_zmq_port)
+            logger.debug("Created monitoring radio")
```

Comment:
I'm sure we've `.debug()`ed the `hub_address` and `hub_zmq_port` elsewhere ... would it make sense to `.debug()` them here as well?  I thinking in terms of an edge case where they, for whatever reason, _don't_ line up with the other log lines.

Reply:
I'd also like to log this inside the ZMQRadioSender itself, but multiprocessing-driven awkward log configuration means that parsl.* logs don't go into the interchange log (or anywhere since #3463 - prior to that they appeared inside the main parsl.log even though they came from the interchange process!)